internal/pkg/manifest/svc.go

```diff
@@ -416,3 +416,35 @@ func ParsePortMapping(s *string) (port *string, protocol *string, err error) {
 		return nil, nil, fmt.Errorf("cannot parse port mapping from %s", *s)
 	}
 }
+
+type fromEnvironment struct {
+	Name *string `yaml:"from_environment"`
```

Comment:
I always forget, do we opt-in to `*string` over `string` due to mergo?

Reply:
yesssss